uber/site_sections/jobs.py

```diff
@@ -224,3 +224,36 @@ def add_volunteers_by_dept(self, session, message='', location=None):
                                 .order_by(Attendee.full_name).all()
             ]
         }
+
+    @ajax
+    def add_bulk_admin_accounts(self, session, ids, message='', **params):
+        if isinstance(ids, str):
+            ids = str(ids).split(",")
+        success_count = 0
+        for id in ids:
+            try:
+                uuid.UUID(id)
+            except ValueError:
+                pass
+            else:
+                match = session.query(Attendee).filter(Attendee.id == id).first()
+                if match:
+                    account = session.admin_account(params, checkgroups=['access'])
+                    if account.is_new:
+                        password = genpasswd()
+                        account.hashed = bcrypt.hashpw(password, bcrypt.gensalt())
+                        account.attendee = match
+                        session.add(account)
+                        body = render('emails/accounts/new_account.txt', {
+                            'account': account,
+                            'password': password
+                        })
+                        send_email(c.ADMIN_EMAIL, match.email, 'New ' + c.EVENT_NAME + ' Ubersystem Account', body)
```

Comment:
Since you had to copy the code, plus check in multiple places for `ACCOUNTS` access, I would say that this may be better placed in the Accounts page. I'm assuming the reason you didn't do this is because there's no mechanism in the Accounts page to list people by department, but I'd also say that making that list appear on the Accounts page would be far better UX than our current "dropdown of literally every staffer in existence."

Also, we want to remove references to "Ubersystem," so let's call this a RAMS account please. :)